scripts/libs.openmpif90_forcxx

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
I struggled a bit with what to name this script and am open to other suggestions. I was trying to distinguish this from the existing `libs.openmpif90` (which is still needed for other compiler combinations): both of them accept the mpif90 wrapper executable, which I think is what's indicated by the name `libs.openmpif90`, but the original produces the libraries needed when linking with the f90 compiler, whereas the new one produces the libraries needed when linking with the C++ compiler.

Reply:
That name looks fine to me. We haven't been super strict about naming convention of these internally used scripts. The name of this one seems as clear as any other.